library/date/next_day_spec.rb

```diff
@@ -0,0 +1,10 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require 'Date'
```

Comment:
It should be `require 'date'` by convention and also because this only works on case-insensitive filesystems.